CHANGES.md

```diff
@@ -1,3 +1,41 @@
+## edge-19.2.1
+
```

Comment:
In previous edge releases with breaking changes, we've included a "Upgrade notes" block here. Is that relevant here? E.g. if someone is using proxy injection already, do they have to do something special to upgrade?

Reply:
The first bullet point with the breaking change includes a link to https://linkerd.io/2/proxy-injection/, which explains the new setup. I think that's probably good enough.

Reply:
Works for me